collect_app/src/main/java/org/odk/collect/android/logic/PropertyManager.java

```diff
@@ -112,12 +92,8 @@ public PropertyManager(Application application, RxEventBus rxEventBus, Permissio
 
     public PropertyManager reload() {
         try {
-            // Device-defined properties
-            IdAndPrefix idp = findDeviceId(context, deviceDetailsProvider);
-            putProperty(PROPMGR_DEVICE_ID,     idp.prefix,          idp.id);
+            putProperty(PROPMGR_DEVICE_ID,     SCHEME_IMEI,         deviceDetailsProvider.getDeviceId());
```

Comment:
What effect does having `SCHEME_IMEI` here have? Seems it should be `collect`, maybe?

Reply:
That's a good question, I wanted to ask about it because I'm not sure... I kept SCHEME_IMEI for now but it's not IMEI anymore so it doesn't make much sense. I could do what you say and add collect: but our new device id is already like `collect:5sf45sf5dsf5d4f` (it starts with collect: always) so a result the value with scheme would be `collect:collect:5sf45sf5dsf5d4f`

Of course depends where we use it because in settings and in a form it would be still just `collect:5sf45sf5dsf5d4f` (because there we get the value without scheme) but during sending a form we use:
```
deviceId = new PropertyManager(Collect.getInstance().getApplicationContext())
                    .getSingularProperty(PropertyManager.withUri(PropertyManager.PROPMGR_DEVICE_ID));
```

what will return doubled collect: 
`collect:collect:5sf45sf5dsf5d4f
`

maybe we should get rid of that prefix collect: which now is a part of device id and add it instead of SCHEME_IMEI so as a result in settings and in a form we would have `5sf45sf5dsf5d4f` but sending forms it would be `collect:5sf45sf5dsf5d4f`?

@lognaturel @seadowg

Reply:
Interesting, I didn't realize there was that difference between in-form and at submission time. I think they should be consistent and both include the scheme. How about just using the literal String `""` as the scheme here?

Reply:
That would be a solution, what do you think @seadowg?

Reply:
Ah yeah I hadn't realized `PropertyManager` expects a scheme here. I think passing `""` as the scheme makes the most sense to me as because then (as @lognaturel points out) it'll be displayed consistently.

We could then just add a `putProperty(String propName, String value)` that doesn't append a scheme (or just calls the other `putProperty` with `""`).

Reply:
Ok, I removed that scheme but I needed to add some extra changes in `InstanceServerUploaderTask` and `InstanceSubmitter` where we access that property to avoid using PropertyManager.withUri because then even if the scheme is empty we would have an extra semicolon left, like: `:collect:5sf45sf5dsf5d4f `.
All the changes are in my last commit:
https://github.com/getodk/collect/pull/4191/commits/94e954c9456be9b5654fc759d80c31bac9da5c28